Register/stylesheet.css

```diff
@@ -0,0 +1,101 @@
+html {
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  height: 100%;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Nunito", sans-serif;
+  color: grey;
+}
+
+#content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  background-image: url("../Images/bg.jpg");
+  margin: 0px;
+}
+
+form {
+  display: flex;
+  width: fit-content;
+  height: fit-content;
+  background: white;
+  flex-direction: column;
+  padding: 16px;
+}
+
+input {
+  background-color: white;
+  width: 200px;
+  margin-bottom: 2em;
+  border: 1px solid rgb(194, 193, 193);
+  border-radius: 6px;
+  width: 400px;
+  padding: 10px;
+}
+label {
+  margin-bottom: 10px;
+  font-size: small;
+  font-weight: bold;
+}
+
+button {
+  cursor: pointer;
+}
+
+h4 {
+  margin: 0px;
+  padding-bottom: 18px;
+  color: black;
+  font-size: large;
+  font-weight: bold;
+}
+
+#switch input {
+  width: max-content;
+}
+
+a {
+  text-decoration: none;
+  padding-left: 8px;
+  color: blue;
+}
+
+#registerBtn {
+  color: white;
+  background-color: rgb(120, 120, 253);
+  border-radius: 7px;
+  height: 30px;
+  border: none;
+}
+
+#registerBtn * {
+  vertical-align: middle;
+  color: white;
+}
+
+#registerBtn span {
+  padding-right: 8px;
+}
+
+#agreeTerms {
+  display: flex;
+  justify-content: center;
+  font-size: small;
+}
+
+#login {
+  font-size: small;
+  display: flex;
+  justify-content: center;
+  margin: 18px;
+}
```

Comment:
Duplicated. Use the same stylesheet used for the login page.

Reply:
fixed